benchmarks/benchmark_run_tardis.py

```diff
@@ -2,19 +2,39 @@
 import os
 from tardis.io.configuration.config_reader import Configuration
 from tardis import run_tardis
+from astropy import units as u
 
 class Benchmarkruntardis:
     """Class to benchmark the run_tardis function.
     """
     timeout = 200
-    
-    def setup(self):
+    params = ['branch85_w7', 'uniform','power_law','exponential']
+    param_names = ['Density']
+
+    def setup(self, d):
         filename = "tardis_configv1_benchmark.yml"
         dir_path = os.path.dirname(os.path.realpath(__file__))
         path = os.path.join(dir_path, "data", filename)
         config = Configuration.from_yaml(path)
         config.atom_data = "kurucz_cd23_chianti_H_He.h5"
+        config.model.structure.density.type = d
+        config.model.structure.velocity.start = 1000 * u.km/u.s
+        config.model.structure.velocity.stop = 2000 * u.km/u.s
+        config.model.structure.velocity.num = 20
+        if d == 'uniform':
+            config.model.structure.density.time_0 = 1 * u.day
```

Comment:
If many of these settings are the same, they don't need to be in the if statements

Reply:
I have added the common settings outside the if clause and anything that differ from default inside the if clause. I thought it would be better for readability.